### PR TITLE
Features/93 backend

### PR DESCRIPTION
**Pull request by bomanviktor**
Backend functionality for avatar and bio.

### CODE REVIEW

**Review by maximihajlov — CHANGES_REQUESTED** [review]
Bra jobbat!👏 

It works well, but there're several comments about good practices. Looks like you've also accidently commented the `/logout` test.

**Comment by maximihajlov** [comment]
I also see that sometimes you push fix commits to your own branch (like f1eadff9edeefcf2166861694cd03a5fc04f7141). Remember that you can always interactively rebase your own branch, squashing the original commit (e.g. `Add 'GetLastUserId'`) and your fix commit (e.g. `Hotfix 'GetLastUserId'`)

And word `Hotfix` is usually used for production patches (that's why it's hot).

File: backend/forum/handlers/test/auth_test.go [hunk 1]
```diff
@@ -13,8 +13,15 @@ func TestSignUp(t *testing.T) {
...
+		client := NewClientData()
+		cli.TestPost(t, "/api/signup", client, http.StatusOK)
+
+		client.Email = "another@email.com"
+		client.Username = ""
+		client.Bio = ""
+		client.Avatar = ""
+		cli.TestPost(t, "/api/signup", client, http.StatusOK)
 
-		cli.TestPost(t, "/api/signup", NewClientData(), http.StatusOK)
```
[the remark this answers is not in the document]
Comment (bomanviktor) [reply backend/forum/handlers/test/auth_test.go:17]: Oh, thanks. Forgot about this!

File: backend/forum/handlers/test/auth_test.go [hunk 2]
```diff
@@ -216,7 +237,7 @@ func TestLogout(t *testing.T) {
 
 	cli := testServer.TestClient()
 
-	cli.TestPost(t, "/api/logout", nil, http.StatusUnauthorized)
+	// cli.TestPost(t, "/api/logout", nil, http.StatusUnauthorized)
```
[the remark this answers is not in the document]
Comment (bomanviktor) [reply backend/forum/handlers/test/auth_test.go:240]: It refused to test the unauthorized error in the logout handler. So I tried different things, and forgot to uncomment.

**Review by maximihajlov — APPROVED** [review]
LGTM 🚀

**Pull request merged**